.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -0,0 +1,176 @@
+# This is a reusbale workflow to bump the 'sec-scanners-config.yaml'
+#
+# It will do so by using the script `hack/scripts/render-sec-scanners-config.sh`, that is not part of the workflow.
+# If you want to run this workflow against a repo the script must exist in that repo. This is by design, because every repo
+# will require a specfic sec-scanners-config.yaml.
+#
+# The script `render-sec-scanners-config.sh` will in all cases require a version that is used to tag the corresponding image
+# of the controller. For this reason, passing the input `version_tag` is required.
+#
+# To create a PR and monitor it, this workflow will require a classic github personal access token (pat) passed
+# as a secret named `BOT_PAT`. The token must be configured to have all rights for `repo`, `user` and `workflow`.
+# Further reads:
+#   Setting a secret for a repo: https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions
+#
+# If changes were done by the script, the workflow will create a PR and wait for it to be merged.
+# The waiting will happen with a timeout that can be set via the input of `timeout`. The units are seconds.
+# It has a default value of 3600 (seconds (= 1 hour)).
```

Comment:
Is there default timeout by github action for a single step?

Reply:
Yes:
```
Job execution time - Each job in a workflow can run for up to 6 hours of execution time. If a job reaches this limit, the job is terminated and fails to complete.
```
https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits
Do you think it needs to be mentioned and/or linked here?